### ISSUE TITLE
yamcs-client does not initialize properly unless importing YamcsClient first

### ISSUE THREAD

**Issue by merose** [yamcs/python-yamcs-client#28]
An application of ours had a submodule that needed to import yamcs.tmtc.client import ContainerSubscription, for strong typing. But importing that alone in yamcs-client 1.9.8 gives an error.

~~~
Python 3.8.10 (default, Mar 25 2024, 10:42:49) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from yamcs.tmtc.client import ContainerSubscription
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/merose/work/venv/yamcs-5.9/lib/python3.8/site-packages/yamcs/tmtc/client.py", line 7, in <module>
    from yamcs.client.activities import ScriptActivity
  File "/home/merose/work/venv/yamcs-5.9/lib/python3.8/site-packages/yamcs/client/__init__.py", line 5, in <module>
    from yamcs.client.core import *  # noqa
  File "/home/merose/work/venv/yamcs-5.9/lib/python3.8/site-packages/yamcs/client/core.py", line 51, in <module>
    from yamcs.tmtc.client import ProcessorClient, _build_value_proto
ImportError: cannot import name 'ProcessorClient' from partially initialized module 'yamcs.tmtc.client' (most likely due to a circular import) (/home/merose/work/venv/yamcs-5.9/lib/python3.8/site-packages/yamcs/tmtc/client.py)
>>>
~~~

Importing YamcsClient first works around it, but the library should initialize properly even if only some classes are imported.

~~~
Python 3.8.10 (default, Mar 25 2024, 10:42:49) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from yamcs.client import YamcsClient
>>> from yamcs.tmtc.client import ContainerSubscription
>>> 
~~~

**Comment by fqqb** [comment]
What would also work is to import like this:

```
from yamcs.client import ContainerSubscription
```

In recent versions, everything has been made available using these shorter imports for simplicity reasons.

But yes, this ImportError can be fixed.

**Comment by fqqb** [comment]
Fixed on master, and in 1.10.0 release
